l10n_be_vat_reports/wizard/l10n_be_partner_vat_listing.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+  <data>
+
+    <menuitem
+        id="menu_finance_belgian_statement"
+        name="Belgium Statements"
+        parent="account.menu_finance_reports" groups="account.group_account_manager"/>
```

Comment:
This menu already exists in l10n_be indeed.

Se example use here: https://github.com/OCA/l10n-belgium/blob/10.0/l10n_be_mis_reports/data/mis_report_vat.xml#L12

Reply:
Thanks.